pkg/webui/components/key-value-map/entry.js

```diff
@@ -143,11 +175,17 @@ Entry.propTypes = {
     PropTypes.string,
   ]),
   valuePlaceholder: PropTypes.message.isRequired,
+  withOptionsUpdate: PropTypes.bool,
 }
 
 Entry.defaultProps = {
   value: undefined,
   readOnly: false,
+  removeMessage: undefined,
```

Comment:
```suggestion
  removeMessage: sharedMessages.remove,
```
Safe to set this as a default.